app/src/main/java/fr/free/nrw/commons/theme/BaseActivity.java

```diff
@@ -27,6 +28,8 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         wasPreviouslyDarkTheme = systemThemeUtils.isDeviceInNightMode();
         setTheme(wasPreviouslyDarkTheme ? R.style.DarkAppTheme : R.style.LightAppTheme);
+        float fontScale = android.provider.Settings.System.getFloat(getBaseContext().getContentResolver(),android.provider.Settings.System.FONT_SCALE, 1f);
```

Comment:
There should be a new line after line 31

Reply:
@Ayan-10 What convention are you referring to?

Reply:
I just thought It will look better and better for readability If a new line will be there.